client/components/popover/style.scss

```diff
@@ -2,46 +2,38 @@
  * "popover" theme for `component/tip`.
  */
 
-// not sure if we need this
-.tip-hide {
-	opacity: 0;
-}
-
-.popover {
+.popover__container {
```

Comment:
I'm getting the eslint-wanring `className should follow CSS name ..._prefix`. I guess it's because `<Popover />` is child of `<RootChild />` component.

![image](https://cloud.githubusercontent.com/assets/77539/17376564/875de97c-598c-11e6-8815-17a9f022bb07.png)

We can easily avoid this warning using `classNames()` function. (In fact it's already used in this PR)